internal/pkg/project/manifest/manifest_test.go

```diff
@@ -4,13 +4,18 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"go.uber.org/zap"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
 	"github.com/keboola/keboola-as-code/internal/pkg/testhelper"
 )
 
+type fakeNaming struct{}
```

Comment:
Naming bude potrebne este tiez rozdelit na viac casti.
Zatial je to spravene trocha provizorne.
Vytovoril som interface `naming` s metodou `Attach` a tu v teste je NOP implementacia.